cpp/src/parquet/properties.h

```diff
@@ -1003,9 +1004,23 @@ class PARQUET_EXPORT ArrowReaderProperties {
   void set_should_load_statistics(bool should_load_statistics) {
     should_load_statistics_ = should_load_statistics;
   }
+
   /// Return whether loading statistics as much as possible.
   bool should_load_statistics() const { return should_load_statistics_; }
 
+  /// Convert undefined logical types as their underlying physical type
+  ///
+  /// When enabled, the Arrow reader will use the underlying physical type to deduce the
+  /// Arrow type when the logical type is not recognized (e.g., one that was added to the
+  /// spec but not implemented in Parquet C++).
+  void set_convert_undefined_logical_types(bool convert_undefined_logical_types) {
```

Comment:
"Convert" is misleading because there is no conversion ongoing, on the contrary.
Perhaps `allow_undefined_logical_types`?